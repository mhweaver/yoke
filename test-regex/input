this is a test file 
second line!
this is a number: 25;

5th lIne.