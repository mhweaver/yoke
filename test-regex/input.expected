this is a test file 
second line!

4th LINE.